### DISCUSSION TITLE
Flagging the two sunset/night beach levels for outside of time attack?

### DISCUSSION THREAD

**Discussion by Tenome** [emoose/OutRun2006Tweaks#257] [Suggestions]
### Describe the feature you'd like to see added

Going off the coattails of the stage transition option, would it be possible to flag the sunset/night beach levels that appear in time attack as having a chance to appear elsewhere? They're exclusive stages, but they're just lighting changes if I recall.

**Comment by envido32** [comment]
I just tested [Howard's Course Editor](https://forum.arcadecontrols.com/index.php/topic,130891.msg1681494.html#msg1681494) and it mostly works, it's a bit buggy specially in "Heart Attack Mode".

You don't need to use FXT, works the same with  emoose's Tweaks (or even with no mod)

You may need to change the Scripts directory properties, uncheck the "Read only" attribute:
![image](https://github.com/user-attachments/assets/faca7fcb-6a2e-41e5-8b1c-3fd3c652ab07)

![OutRun2006_ Coast to Coast 28_7_2024 20_30_07](https://github.com/user-attachments/assets/75ea6fb6-8464-4b45-b510-45c4a1f88f6e)

**Comment by Tenome** [comment]
Oh, yeah, I'm aware that you can force it with the course editor, I was just hoping that it wouldn't be needed since that locks the race into that specific ordering (since it replaces the script bin file).

**Comment by envido32** [comment]
It looks possible in principle, I was reading the thread and it seems that there was a more feature rich version in development but was not released. It even randomized the course on the fly and could have endless loops.

It seems that it was quite hard to debbug because there are a lot of addresses pointers and to test it you need to run the courses again and again.

I took a look at the posted data, but it's above my expertise... so I'm not sure how hard could it actually be.

**Comment by Tenome** [comment]
A working endless mode would be sweet. I think the course editor allows you to set the first track as one of the exit points in a 15-course, but I don't remember if it actually works.
